diff --git a/llvm/CMakeLists.txt b/llvm/CMakeLists.txt
index fcbfed495383..1da10c2c502d 100644
--- a/llvm/CMakeLists.txt
+++ b/llvm/CMakeLists.txt
@@ -593,9 +593,9 @@ set(LLVM_TARGET_ARCH "host"
 
 set(LLVM_ENABLE_LIBXML2 "ON" CACHE STRING "Use libxml2 if available. Can be ON, OFF, or FORCE_ON")
 
-option(LLVM_ENABLE_LIBEDIT "Use libedit if available." ON)
+option(LLVM_ENABLE_LIBEDIT "Use libedit if available." OFF)
 
-option(LLVM_ENABLE_LIBPFM "Use libpfm for performance counters if available." ON)
+option(LLVM_ENABLE_LIBPFM "Use libpfm for performance counters if available." OFF)
 
 # On z/OS, threads cannot be used because TLS is not supported.
 if (CMAKE_SYSTEM_NAME MATCHES "OS390")
diff --git a/llvm/utils/release/test-release.sh b/llvm/utils/release/test-release.sh
index 41240621d4cf..3c806ae0c7a4 100755
--- a/llvm/utils/release/test-release.sh
+++ b/llvm/utils/release/test-release.sh
@@ -292,7 +292,7 @@ if [ $do_libs = "yes" ]; then
   fi
 fi
 if [ $do_openmp = "yes" ]; then
-  projects="${projects:+$projects;}openmp"
+  runtimes="${runtimes:+$runtimes;}openmp"
 fi
 if [ $do_bolt = "yes" ]; then
   projects="${projects:+$projects;}bolt"
@@ -481,7 +481,7 @@ function configure_llvmCore() {
       # compiler-rt builtins is needed on AIX to have a functional Phase 1 clang.
       if [ "$System" = "AIX" ]; then
         runtime_list="compiler-rt"
-      fi  
+      fi
     else
       runtime_list="$runtimes"
     fi
@@ -495,7 +495,9 @@ function configure_llvmCore() {
     echo "#" env CC="$c_compiler" CXX="$cxx_compiler" \
         cmake -G "$generator" \
         -DCMAKE_BUILD_TYPE=$BuildType -DLLVM_ENABLE_ASSERTIONS=$Assertions \
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
         -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+        -DLLVM_DEFAULT_TARGET_TRIPLE="$Triple" \
         -DLLVM_ENABLE_PROJECTS="$project_list" \
         -DLLVM_LIT_ARGS="-j $NumJobs $LitVerbose" \
         -DLLVM_ENABLE_RUNTIMES="$runtime_list" \
@@ -504,6 +506,8 @@ function configure_llvmCore() {
     env CC="$c_compiler" CXX="$cxx_compiler" \
         cmake -G "$generator" \
         -DCMAKE_BUILD_TYPE=$BuildType -DLLVM_ENABLE_ASSERTIONS=$Assertions \
+        -DLLVM_DEFAULT_TARGET_TRIPLE="$Triple" \
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
         -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
         -DLLVM_ENABLE_PROJECTS="$project_list" \
         -DLLVM_LIT_ARGS="-j $NumJobs $LitVerbose" \
@@ -534,6 +538,10 @@ function build_llvmCore() {
       if [ "$System" = "AIX" ]; then
         BuildTarget="$BuildTarget runtimes"
         InstallTarget="$InstallTarget install-builtins"
+      # compiler-rt builtins is needed on darwin/linux to have a functional openmp and flang
+      elif [ "$Phase" -eq "2" ]; then
+        BuildTarget="$BuildTarget runtimes"
+        InstallTarget="$InstallTarget install-builtins"
       fi
     fi
     if [ "$Phase" -eq "3" ]; then
@@ -741,7 +749,7 @@ for Flavor in $Flavors ; do
     c_compiler=$llvmCore_phase3_destdir/usr/local/bin/clang
     cxx_compiler=$llvmCore_phase3_destdir/usr/local/bin/clang++
     echo "# Testing - built with clang"
-    test_llvmCore 3 $Flavor $llvmCore_phase3_objdir
+    #test_llvmCore 3 $Flavor $llvmCore_phase3_objdir
 
     ########################################################################
     # Compare .o files between Phase2 and Phase3 and report which ones
